.travis.yml

```diff
@@ -1,40 +1,46 @@
 language: php
 
-cache: vendor
```

Comment:
ok travis is clever:

``
$ export CODECLIMATE_REPO_TOKEN=[secure]
$ export GITHUB_OAUTH_TOKEN=[secure]
``